webapp/views/wpt-results.js

```diff
@@ -855,6 +979,132 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
     }
   }
 
+  shouldDisplayHarnessWarning(node, index) {
+    // Determine if a warning sign should be displayed next to subtest counts.
+    const status = node.results[index].status;
+    return !node.isDir && status && !PASSING_STATUSES.includes(status)
+      && !node.results.every(testInfo => testInfo.singleSubtest);
+  }
+
+  getStatusDisplay(node, index) {
+    let status = node.results[index].status;
+    if (status in STATUS_ABBREVIATIONS) {
+      status = STATUS_ABBREVIATIONS[status];
+    }
+    return status;
```

Comment:
Quick question: In the future, is there a possibility where an additional abbreviation is added to the results processor STATUS_ABBREVIATIONS but has not been added to STATUS_ABBREVIATIONS here. Do we want to add an else conditional that defaults to something like `"Unknown Status: " + status`? Or could the browser cache this version of the javascript while the results processor has added a new abbreviation?